texttmining.py

```diff
@@ -0,0 +1,106 @@
+import requests
+import nltk
+import pickle
+from bs4 import BeautifulSoup
+
+#Mary_Wade_text = requests.get('http://www.gutenberg.org/cache/epub/43585/pg43585.txt').text
+#print(Mildred_Wirt_text)
+
+# Save data to a file (will be part of your data fetching script)
+# f = open('Mary_Wade_text.pickle', 'wb')
+# pickle.dump(Mary_Wade_text, f)
+# f.close()
+
+#Load data from a file (will be part of your data processing script)
+def get_pickle(file_name):
+    input_file = open(file_name, 'rb')
+    #input_file.seek(0)
+    reloaded_copy_of_texts = pickle.load(input_file)
+    reloaded_copy_of_texts = reloaded_copy_of_texts.split("END OF THIS PROJECT GUTENBERG EBOOK")
+    return reloaded_copy_of_texts[0]
+# #soup = BeautifulSoup(Wilkie_Collins_text, 'html.parser')
+#print(soup.prettify())
+
+def word_sectioning(s):
+    """ Takes a string of words returns the first word
+
+        s: a string
+        returns: the first word
+    >>> word_sectioning("I bat today")
+    'I'
+    >>> word_sectioning("I.ban")
+    'I'
+    """
+    for index in range(0,len(s)):
+        word_section = s[index]
+        if  word_section == ' ' or word_section == '.' or  word_section == ',' or  word_section == '?' or  word_section == '!' or  word_section == ';' or  word_section == ':':
+#        if  word_section == ' ':
+            return s[:index]
+    return s
+
+def word_finder(s):
+    """ Takes a string of words returns all of the words. Does not work if the
+    sentence begings with a "stopper"
+
+        s: a string
+        returns: all of the word
+    >>> word_sectioning(".I ran back. to the bat.I")
+    'I', 'ran', 'back', 'to', 'the', 'bat', 'I'
+    """
+    index = 0
+    word_start_stop_list = []
+    s = ' ' + s
+    while index+1 < len(s):
+        if ((s[index] == ' ' or  s[index]== '.' or  s[index] == ',' or  s[index] == '?' or  s[index] == '!' or  s[index] == ';' or  s[index] == ':')
```

Comment:
More concise way to phrase this condition is by doing something like this:
`if s[index] in ' .,?!;:'`